### DISCUSSION TITLE
"Error moving pdb file!" Error when running EOM

### DISCUSSION THREAD

**Discussion by orionshih** [biosaxs-com/atsas-community#1951] [Mixtures and flexible systems] [answered]
Dear all,

I was submitting an EOM calculation on the ATSAS web interface to test something I'd already tested at the beginning of September. After I clicked "SUBMIT," the system took me to a page that showed "Error moving pdb file!" and didn't proceed with the calculation...but I ran the same calculation at the beginning of September without any error. (calculation results attached below)
[0905t1.zip](https://github.com/user-attachments/files/17161445/0905t1.zip)

Has something changed from Sept 5 til now...? Any help/information would be greatly appreciated. Thank you!

Orion

**Comment by orionshih** [comment]
Hi,

I saw the same error when I tried to run a CORAL calculation as well...(as shown below)
![image](https://github.com/user-attachments/assets/14f3958c-9db3-43e7-a264-dce622e31699)

Does ATSAS add some restrictions on PDB files or...?
Thanks..

Orion

**Comment by franke-biosaxs** [comment]
ATSAS online currently experiences technical difficulties. IT support has been informed.

**Comment by ClementB-EMBL** [comment, marked as answer]
Hi Orion,

ATSAS online is back up. Please try again.

Clement
